### PR TITLE
adding patool

### PR DESCRIPTION
**Pull request by notestaff**
Needed as a prerequisite for DataLad .

### CODE REVIEW

File: recipes/patool/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+{% set version = "1.12" %}
+{% set sha256 = "e3180cf8bfe13bedbcf6f5628452fca0c2c84a3b5ae8c2d3f55720ea04cb1097" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/1b/eb/ad3c94cb8cbc1d8b1c47471d2c43537a05fda2bdff54a7d8248873591691/{{ name }}-{{ version }}.tar.gz
```
Comment (isuruf) [inline recipes/patool/meta.yaml]: Use the url from pypi as in the example recipe.

File: recipes/patool/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
```
Comment (synapticarbors) [inline recipes/patool/meta.yaml]: It looks like `patools` can be used from a command line interface. Could you add a `commands` test for this:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-commands

File: recipes/patool/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  imports:
+    - patoolib
+    - patoolib.programs
+
+about:
+  home: http://wummel.github.io/patool/
+  license: GPLv3
```
Comment (synapticarbors) [inline recipes/patool/meta.yaml]: Change `GPLv3` to `GPL-3.0`:

https://opensource.org/licenses/alphabetical

**Comment by synapticarbors** [comment]
I'm not sure if it's valid to have `noarch: python` package that has a custom linux/osx only build script (if only just for testing). It looks like the `patools` script is built on top of `argparse`, so you should be able to just test it's existence with the `-h` flag. Or is there another issue?

**Comment by synapticarbors** [comment]
I believe that you should return to building on all systems using `noarch: python`. Is there a reason why you need to use the `build.sh` instead of relying on the `script` entry in the `meta.yaml` as you did before?

**Comment by notestaff** [comment]
@synapticarbors Since there is a command-line part, getting this to work correctly with Windows batch files requires additional work.  It would help me if I could get patools into conda-forge for Linux and MacOS now, and maybe look into a Windows version later.  I'm adding patools because it's a dependency of the package I actually want to add, DataLad .  I recall reading somewhere that a Windows build is optional for conda-forge?

**Comment by synapticarbors** [comment]
@conda-forge/staged-recipes -- does anyone have any ideas about why the `patool` script, which is listed in the `setup.py` under `scripts` is having permission issues? See the error here:

https://circleci.com/gh/conda-forge/staged-recipes/20905

which was generated by this alternative PR:

https://github.com/conda-forge/staged-recipes/pull/4233

**Comment by notestaff** [comment]
@conda-forge/core Just checking if someone could look at this PR, when have time.

**Comment by synapticarbors** [comment]
It looks good to me, although I still wonder about the question I raised in:
https://github.com/conda-forge/staged-recipes/pull/4206#issuecomment-339327307

I'd still like @conda-forge/staged-recipes to weigh in on that, although perhaps it can be addressed in the feedstock.

**Comment by scopatz** [comment]
Hmm that is odd @synapticarbors. Please address it in the feedstock, if it becomes a problem. No need to hold this up longer.

**Pull request merged**